gnomad_qc/v4/sample_qc/identify_trios.py

```diff
@@ -134,15 +167,47 @@ def run_mendel_errors(
         vds = hl.vds.filter_chromosomes(vds, keep="chr20")
 
     vds = hl.vds.filter_samples(vds, ped_samples)
+    vds.variant_data = vds.variant_data.select_entries("LA", "LGT")
     mt = hl.vds.to_dense_mt(vds)
-    mt = mt.select_entries("GT")
+
+    if interval_qc_pass_ht is not None:
+        mt = mt.filter_rows(interval_qc_pass_ht[mt.locus].pass_interval_qc)
+
+    mt = hl.experimental.sparse_split_multi(mt, filter_changed_loci=True)
 
     logger.info(f"Running Mendel errors for %s trios.", len(merged_ped.trios))
     mendel_err_ht, _, _, _ = hl.mendel_errors(mt["GT"], merged_ped)
 
     return mendel_err_ht
 
 
+def filter_ped_to_same_platform(
+    ped: hl.Pedigree,
+    platform_ht: hl.Table,
+) -> hl.Pedigree:
+    """
+    Filter a Pedigree to only trios where all samples are from the same platform.
+
+    :param ped: Pedigree to filter.
+    :param platform_ht: Table with platform assignments.
+    :return: Filtered Pedigree.
+    """
+    sample_to_platform = platform_table_to_dict(platform_ht)
+
+    filtered_trios = []
+    for trio in ped.trios:
+        trio_ids = [trio.s, trio.pat_id, trio.mat_id]
+        if len({sample_to_platform[s] for s in trio_ids}) == 1:
+            filtered_trios.append(trio)
+
+    logger.info(
+        "Found %i trios with all samples having the same platform assignment.",
```

Comment:
could be helpful to add out of how many trios were looked at here to get a sense of what proportion were kept